Packages/manifest.json

```diff
@@ -2,6 +2,7 @@
   "dependencies": {
     "com.unity.2d.sprite": "1.0.0",
     "com.unity.ide.rider": "3.0.18",
+    "com.unity.services.authentication": "2.5.0-pre.3",
     "com.unity.services.friends": "0.2.0-preview.9",
```

Comment:
Is this the latest public version of the friends SDK?

Reply:
It's the minimum value required for player names, so I believe so. I'll double check.

Reply:
Oh sorry, I thought you said auth SDK. As far as I know this is the most recent friends package, yes.